apis/python/src/tiledbsc/scgroup.py

```diff
@@ -234,7 +234,7 @@ def write_X(self, anndata: ad.AnnData):
 
         dom = tiledb.Domain(
             tiledb.Dim(name="obs_id", domain=(None, None), dtype="ascii", filters=[tiledb.RleFilter()]),
-            tiledb.Dim(name="var_id", domain=(None, None), dtype="ascii", filters=[tiledb.ZstdFilter()]),
+            tiledb.Dim(name="var_id", domain=(None, None), dtype="ascii", filters=[tiledb.ZstdFilter(level=22)]),
```

Comment:
`level=22` might incur significant overhead/slowness for decompression aka reads. I'd want us to check this before we merge this. We might over trade ondisk size compared to read performance.

Reply:
ok thanks! this is a shameless copy-paste of CZI's 'current best' but it's not strictly necessary to use that here _as a default_ --_ soon (next day or two, 🤞) we'll support exposing tiledb schema parameters to the `tiledbsc-r` user via args in the soma-constructor -- this can wait.

Reply:
we found it was a good trade where the data was on S3 (which is normal for us) - we just threw a lot of CPU at it and it was net postive